### ISSUE TITLE
Pentadactyl sometimes gives error when using various methods of scrolling the page

### ISSUE THREAD

**Issue by doomikus** [5digits/dactyl#80]
I receive this message at least some of the time when using C-d, C-u, PageUp, PageDown and possibly others (G and gg usually seem to work):

/resource://dactyl/buffer.jsm 1680: TypeError: options is undefined

my .pentadactylrc:

"hg7224

loadplugins '\.(js|penta)$'
group user
set cdpath=''

" vim: set ft=pentadactyl:

**Comment by haifengkao** [comment]
got the same problem

**Comment by PaBLoX-CL** [comment]
I found that I workaround is just open pentadactyl and close it... yes, totally windows like xD. 

<kbd>:</kbd><kbd>Esc</kbd>

**Comment by minego** [comment]
I'm seeing the same issue. What do you mean "just open pentadactyl and close it"? Does that mean open the command prompt with :, or restart firefox?

**Comment by krompus** [comment]
I think they mean "restart Penta".

There are two ways to do this: `:rehash`, or just restart Firefox entirely, which, in my experience, is a little faster.

**Comment by minego** [comment]
Ah, thank you.

**Comment by krompus** [comment]
Cheers!

**Comment by PaBLoX-CL** [comment]
No, I actually meant open the command line with `:`, and then exit with `esc`

**Comment by krompus** [comment]
Ah, gotcha.

**Comment by krompus** [comment]
arst

**Comment by polyzen** [comment]
Duplicate of #12? If so, apparently the workaround no longer works.
